{"name": "test_input[field_data_dict0]", "status": "broken", "statusDetails": {"message": "TypeError: ContactForm.input_field() takes 2 positional arguments but 4 were given", "trace": "field_data_dict = {'email': 'johndoe@example.com', 'message': 'This is my message', 'name': 'John Doe', 'phone': '1234567890'}\n\n    @pytest.mark.parametrize(\"field_data_dict\", [{\n            \"name\": \"John Doe\",\n            \"phone\": \"1234567890\",\n            \"email\": \"johndoe@example.com\",\n            \"message\": \"This is my message\"\n        }])\n    def test_input(field_data_dict):\n        contact_page = ContactForm(driver)\n>       contact_page.input_field(driver, \"Your Name*\",field_data_dict)\nE       TypeError: ContactForm.input_field() takes 2 positional arguments but 4 were given\n\ntests\\test_contact_form.py:32: TypeError"}, "parameters": [{"name": "field_data_dict", "value": "{'name': 'John Doe', 'phone': '1234567890', 'email': 'johndoe@example.com', 'message': 'This is my message'}"}], "start": 1708227273159, "stop": 1708227273159, "uuid": "3d6a5161-3f93-4cb9-a10d-c748e143460e", "historyId": "e664e86f401fcbe7f878a69934a3c11e", "testCaseId": "938ec7942806b96867ac01ccf953ed7f", "fullName": "tests.test_contact_form#test_input", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_contact_form"}, {"name": "host", "value": "udav_comp"}, {"name": "thread", "value": "24640-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_contact_form"}]}